sopel_waifu/waifu.json

```
@@ -1511,11 +1560,28 @@
         "Yowane Haku"
     ],
     "WataMote": [
-        "Tomoko Kuroki"
+        "Kuroki Tomoko",
+        "Naruse Yuu",
+        "Satozaki Kiko",
+        "Nemoto Hina",
+        "Megumi Imae",
+        "Ogino",
+        "Akane Okada"
+    ],
+    "White Album":[
```

Obviously I haven't set up a JSON linter yet, but… S T Y L E

```suggestion
    "White Album": [
```